packages/@lightningjs/ui-components/src/components/Column/Column.mdx

```diff
@@ -57,7 +53,7 @@ class BasicExample extends lng.Component {
 }
 ```
 
-The `Column` component extends [FocusManager][fm-docs], so you need to make sure that `_getFocused` returns the `Column` component.
+The `Column` component extends FocusManager, so you need to make sure that `_getFocused` returns the `Column` component.
```

Comment:
Could you add a link to the FocusManager here? I assume that's what "fm-docs" was supposed to be.

Reply:
Do we still want to add this link in?